typography.ftd

```diff
@@ -24,47 +21,76 @@ role: $inherited.types.heading-large
 
 
 -- component heading-medium:
-module design-system: ds
-caption text:
+caption or body text:
 ftd.color color: $inherited.colors.text-strong
 
 -- ftd.text: $heading-medium.text
-width: fill-container
 color: $heading-medium.color
 role: $inherited.types.heading-medium
 
 -- end: heading-medium
 
 
 -- component heading-small:
-module design-system: ds
-caption text:
+caption or body text:
 ftd.color color: $inherited.colors.text-strong
 
 -- ftd.text: $heading-small.text
-width: fill-container
 color: $heading-small.color
 role: $inherited.types.heading-small
 
 -- end: heading-small
 
 
 -- component heading-tiny:
-module design-system: ds
-caption text:
+caption or body text:
 ftd.color color: $inherited.colors.text-strong
 
 -- ftd.text: $heading-tiny.text
-width: fill-container
 color: $heading-tiny.color
 role: $inherited.types.heading-tiny
 
 -- end: heading-tiny
 
 
+-- component button-large:
+caption or body text:
+ftd.color color: $inherited.colors.text-strong
+ftd.display display: inline
+
+-- ftd.text: $button-large.text
+color: $button-large.color
+role: $inherited.types.button-large
+
+-- end: button-large
+
```

Comment:
Button text can never be added independent of button, so remove all such components